pyproject.toml

```diff
@@ -11,7 +11,7 @@ pattern = "default-unprefixed"
 [tool.poetry.dependencies]
 python = "^3.8"
 
-neptune-api = "^0.8.0"
+neptune-api = { git = "https://github.com/neptune-ai/neptune-api.git", rev = "c4bf4b8869fa0065d263c87e58274bb1ea29966c" }
```

Comment:
This should point to a released version, so we need to wait for the PR in `neptune-api` to be merged I guess